### DISCUSSION TITLE
Rhino 8 Folder problem, not working even after restart

### DISCUSSION THREAD

**Discussion by SpaceArchitecture** [Tunny-gh/Tunny_in_Open_Sea#358] [General]
![A](https://github.com/user-attachments/assets/5b4268b8-fcbc-4f9c-a815-47db1dd20df0)
(8.11.24254.15001, 2024-09-10)
Commercial
0.12.0
Windows 10
Multiple restarts, no solution

**Comment by hrntsm** [comment]
Can you send me the result files of the optimization?
Seems to be a loading error in that file.

**Comment by hrntsm** [comment]
I don't understand "Rhino 8 Folder problem" mean. What does it means?
If the problem is with the folder that Rhino8 is using, then Tunny is probably not involved.
